src/metabase/models/dashboard_card.clj

```diff
@@ -363,35 +362,25 @@
            (serdes/infer-self-path "DashboardCard" dashcard)]))
 
-(defmethod serdes/load-xform "DashboardCard"
-  [dashcard]
-  (-> dashcard
-      ;; Deliberately not doing anything to :series, they get handled by load-one! below
-      (dissoc :serdes/meta)
-      (update :card_id                serdes/*import-fk* :model/Card)
-      (update :action_id              serdes/*import-fk* :model/Action)
-      (update :dashboard_id           serdes/*import-fk* :model/Dashboard)
-      (update :dashboard_tab_id       serdes/*import-fk* :model/DashboardTab)
-      (update :created_at             #(if (string? %) (u.date/parse %) %))
-      (update :parameter_mappings     serdes/import-parameter-mappings)
-      (update :visualization_settings serdes/import-visualization-settings)))
-
-(defn- dashboard-card-series-xform
-  [ingested]
-  (-> ingested
-      (update :card_id          serdes/*import-fk* :model/Card)
-      (update :dashboardcard_id serdes/*import-fk* :model/DashboardCard)))
-
-(defmethod serdes/load-one! "DashboardCard"
-  [ingested maybe-local]
-  (let [dashcard ((get-method serdes/load-one! :default) (dissoc ingested :series) maybe-local)]
-    ;; drop all existing series for this card and recreate them
-    ;; TODO: this is unnecessary, but it is simple to implement
-    (t2/delete! :model/DashboardCardSeries :dashboardcard_id (:id dashcard))
-    (doseq [[idx single-series] (map-indexed vector (:series ingested))] ;; a single series has a :card_id only
-      ;; instead of load-one! we use load-insert! here because :serdes/meta isn't necessary because no other
-      ;; entities depend on DashboardCardSeries
-      (serdes/load-insert! "DashboardCardSeries" (-> single-series
-                                                     (assoc :dashboardcard_id (:entity_id dashcard)
-                                                            :position idx)
-                                                     dashboard-card-series-xform)))))
+(defmethod serdes/make-spec "DashboardCard" [_model-name opts]
+  {:copy      [:col :entity_id :row :size_x :size_y]
+   :skip      []
+   :transform {:created_at             (serdes/date)
+               :dashboard_id           (serdes/parent-ref)
+               :card_id                (serdes/fk :model/Card)
+               :action_id              (serdes/fk :model/Action)
+               :dashboard_tab_id       (serdes/fk :model/DashboardTab)
+               :parameter_mappings     {:export serdes/export-parameter-mappings
+                                        :import serdes/import-parameter-mappings}
+               :visualization_settings {:export serdes/export-visualization-settings
+                                        :import serdes/import-visualization-settings}
+               :series
+               (-> (serdes/nested :model/DashboardCardSeries :dashboardcard_id
+                                  (assoc opts
```

Comment:
I've been thinking of making spec "static" (as in - instead of being returned from `make-spec`, make it a top-level map) and dropping "opts" whatsoever (maybe passing them as parameters to `import/export`) — this would also prevent mistakes like I had in `Database.details` before a test and a fix.

Reply:
yep, that would work too

Reply:
but this also depends on having one codepath instead of two to update all export/import functions...